wg-advisors/README.md

```diff
@@ -102,4 +109,38 @@ the ASF. Trust is transitive - that is, you may not know all members
 personally, but each member was nominated and elected by people that
 you, in turn, nominated and elected.
 
+### How to initiate PMC <-> Advisor relationship?
```

Comment:
According to the original mailing list thread of this WG, it doesn't aim at creating a new office in the foundation.

So I assume that there is no special relationship between the Advisor with other office at least at the "advising" level.

Instead, the advisor WG working on the guidelines and generating knowledge. Every advisor WG member or anyone, as individuals, applying this guidelines to advice projects.

Reply:
Yep. That's how I see it as well (and I think previous paragraphs describe it well) - that this is a group of peer ASF members that help other PMCs  and gathers information/exchanges knowledge how various PMC are interpreting and more importantly "executing" their vision of  "Apache Way" - and advise on making it as common as possible where needed **with** the open mind and accepting the fact that there are variations among projects.

Reply:
I added a paragraph clarifying this, I hope it's ok. I also separated out a Board involvement. We've been discussing that with @rbowen a lot on how the Board <-> PMC <-> Advisors triangle relation should look like. I **think** I captured all the aspects of it in this iteration.